script/upgrade/UpdateDefaultStoryNFTTemplate.s.sol

```diff
@@ -48,7 +41,6 @@ contract UpdateDefaultStoryNFTTemplate is UpgradeHelper {
         IStoryNFTFactory(storyNftFactoryAddr).setDefaultStoryNftTemplate(address(newDefaultStoryNftTemplate));
         console2.log("New DefaultStoryNFTTemplate: ", address(newDefaultStoryNftTemplate));
         _postdeploy("DefaultStoryNftTemplate", address(newDefaultStoryNftTemplate));
-        _writeAddress("OrgNFT", orgNftAddr);
-        _writeAddress("StoryNFTFactory", storyNftFactoryAddr);
```

Comment:
It might be can be further simplify with following logic:
```solidity
defaultStoryNftTemplateAddr = address(newDefaultStoryNftTemplate));
_writeAddresses()
```

Reply:
Resolved in [d6393b0](https://github.com/storyprotocol/protocol-periphery-v1/pull/92/commits/d6393b01e60b7d5ca7805b8635fc5a1f20a1b0b8) :)